autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -388,18 +423,20 @@ def evaluate(self, test_loader: torch.utils.data.DataLoader, epoch: int,
                         epoch * len(test_loader) + step,
                     )
 
+        self._scheduler_step(step_interval=StepIntervalUnit.valid, loss=loss_sum / N)
```

Comment:
why do we need this?

Reply:
It is from [ReduceLROnPlateau](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#torch.optim.lr_scheduler.ReduceLROnPlateau).
It says step should be performed after validation.

Reply:
oh, I didn't know that. Thanks.